client/my-sites/site-settings/form-general.jsx

```diff
@@ -377,38 +402,46 @@ export class SiteSettingsFormGeneral extends Component {
 						) }
 					</FormSettingExplanation>
 				</FormLabel>
-				{ ! isNonAtomicJetpackSite && (
-					<>
-						<FormLabel className="site-settings__visibility-label is-private">
-							<FormRadio
-								name="blog_public"
-								value="-1"
-								checked={ -1 === blogPublic && ! wpcomComingSoon }
-								onChange={ () =>
-									this.handleVisibilityOptionChange( {
-										blog_public: -1,
-										wpcom_coming_soon: 0,
-									} )
-								}
-								disabled={ isRequestingSettings }
-								onClick={ eventTracker( 'Clicked Site Visibility Radio Button' ) }
-								label={ translate( 'Private' ) }
-							/>
-						</FormLabel>
-						<FormSettingExplanation isIndented>
-							{ hasLocalizedText(
-								'Your site is only visible to you and logged-in members you approve. Everyone else will see a log in screen.'
-							)
-								? translate(
-										'Your site is only visible to you and logged-in members you approve. Everyone else will see a log in screen.'
-								  )
-								: translate(
-										'Your site is only visible to you and logged-in members you approve.'
-								  ) }
-						</FormSettingExplanation>
-					</>
-				) }
-			</FormFieldset>
+			</Fragment>
+		);
+	}
+
+	privacyRadioComingSoon() {
+		const { fields, isRequestingSettings, eventTracker, translate } = this.props;
+		const wasComingSoon = parseInt( fields.was_wpcom_coming_soon, 10 ) === 1;
+		const isComingSoon = parseInt( fields.wpcom_coming_soon, 10 ) === 1;
+
+		// If v2 and not currently coming soon, hide
```

Comment:
Just to elaborate on the comment and get it straight in my mind, if:

1. coming soon v2 is activated and 
2. the user's site is **not** running v1 mode 

we remove the ability to toggle coming soon altogether. This includes new sites.

Reply:
I think - Yes. v2 sites are just public sites with a behind the scenes site option.

Reply:
Also - iirc during prelaunch these settings are disabled.